src/chess/board.cc

```diff
@@ -427,10 +427,6 @@ void InitializeMagicBitboards() {
                     kBishopDirections);
 }
 
-BitBoard ChessBoard::pawns() const { return pawns_ & kPawnMask; }
-
-BitBoard ChessBoard::en_passant() const { return pawns_ - pawns(); }
```

Comment:
`pawns_ & ~(pawns_ & kPawnMask)` is the same as `pawns_ & (~pawns_ | ~kPawnMask)`, and ~pawns never does anything. simplified to `pawns_ & ~kPawnMask`, or `pawns_ - kPawnMask`